lib/systems/platforms.nix

```diff
@@ -469,17 +468,27 @@ rec {
     '';
   };
 
-  selectBySystem = system: {
-      i486-linux = pc32;
-      i586-linux = pc32;
-      i686-linux = pc32;
-      x86_64-linux = pc64;
-      armv5tel-linux = sheevaplug;
-      armv6l-linux = raspberrypi;
-      armv7a-linux = armv7l-hf-multiplatform;
-      armv7l-linux = armv7l-hf-multiplatform;
-      aarch64-linux = aarch64-multiplatform;
-      mipsel-linux = fuloong2f_n32;
-      powerpc64le-linux = powernv;
-    }.${system} or pcBase;
+  # Deprecated 2020-11
+  selectBySystem = system:
+    lib.warn "`selectBySystem` is deprecated use `lib.systems.platforms.select`"
```

Comment:
I wonder if anything is using this? Might want to do a quick grep of nixpkgs and other places to make sure we aren't accidentally introducing new warnings. I'd actually prefer if these are internal interfaces that can't be reached outside of nixpkgs (via let bindings or whatever).

Reply:
That's fine with me. I doubt it's used either.

Reply:
It's now removed